site/content/docs/cli/command-line-reference.md

```diff
@@ -134,6 +135,7 @@ they contain all the specified tags. Multiple `--tags` flags can be passed, in w
 match any of the `--tag`s filters, i.e. `--tags production,webapp --tags staging,backend` will run checks with tags
 (production AND webapp) OR (staging AND backend).
 - `--test-session-name` A name to use when storing results in Checkly with `--record`.
+- `--retries`: How many times to retry a failing check run.
```

Comment:
```suggestion
- `--retries`: How many times to retry a failing check run. 0 by default and max: 3.
```
would this make sense to add here?

Reply:
This makes sense, good point. I'll update it.

I also just noticed the default for `--timeout` isn't up-to-date anymore - it's now 600 seconds and not 240s. I'll go ahead and update that as well.

One small thing is that if we ever change it, the default will depend on the CLI version the user is on. Users with an old CLI version will have the old default, users with the new CLI will have the new default. 

For `--timeout` users on old CLI versions will still have the default of 240. I think we can take the approach of documenting the default in the most recent CLI version.